mage_ai/settings/__init__.py

```diff
@@ -47,3 +47,23 @@ def is_disable_pipeline_edit_access():
 
 DEFAULT_LOCALHOST_URL = 'http://localhost:6789'
 MAGE_PUBLIC_HOST = os.getenv('MAGE_PUBLIC_HOST') or DEFAULT_LOCALHOST_URL
+
+
+MAGE_SETTINGS_ENVIRONMENT_VARIABLES = [
```

Comment:
Could you add comment for this list so that other developers know what environment variable should be in this list?